code/modules/antagonists/cult/cult_items.dm

```diff
@@ -286,11 +286,22 @@
 	breakouttime = 60
 	knockdown = 30
 
-/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
+#define CULT_BOLA_PICKUP_STUN 6 SECONDS
+/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/carbon/user, list/modifiers)
 	. = ..()
+
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>The bola seems to take on a life of its own!</span>")
-		ensnare(user)
+		if(iscarbon(user))
+			var/mob/living/carbon/C = user
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Rename this `carbon_user` or similar.